README.md

```diff
@@ -1,2 +1,87 @@
 # isar-turtlebot
-ISAR implementation for the Turtlebot3 Waffle Pi
+ISAR implementation for the Turtlebot3 Waffle Pi.
+
+# Installation for development
+
+For local development, please fork the repository. Then, clone and install in the repository root folder:
+
+```bash
+$ git clone https://github.com/equinor/isar-turtlebot
+$ cd isar-turtlebot
+$ pip install -r requirements.txt -r requirements-dev.txt
+$ pip install -e .
+```
+# Simulation prerequisities
+
+The simulator requires a computer or a virtual machine running Ubuntu 20. To install and run the turtlebot simulator, you will need to [install ROS noetic](http://wiki.ros.org/noetic/Installation/Ubuntu) on your computer. The `desktop` version should be sufficient.
+
+
+Then, [install dependent ROS packages](https://emanual.robotis.com/docs/en/platform/turtlebot3/quick-start/), and the [simulation package](https://emanual.robotis.com/docs/en/platform/turtlebot3/simulation/).
+
+You also need to install gazebo:
```

Comment:
Forslag:
"You also need to install gazebo with a [version corresponding](http://gazebosim.org/tutorials/?tut=ros_wrapper_versions) to your ROS distribution"